src/nl/hannahsten/texifyidea/inspections/latex/LatexFileNotFoundInspection.kt

```diff
@@ -109,14 +135,14 @@ open class LatexFileNotFoundInspection : TexifyInspectionBase() {
                     val extension = if (command.commandToken.text in includeOnlyExtensions.keys) {
                         includeOnlyExtensions[command.commandToken.text]?.toList()?.first() ?: "tex"
                     }
-                    else {
-                        "tex"
-                    }
+                    else "tex"
+
+                    val parameterOffset = parameter.text.trimRange(1, 1).indexOf(filePath)
+                    val endoffset: Int = if (filePath.length > 15) 15 else filePath.length
```

Comment:
Why is endOffset only allowed to be max 15?

Reply:
was a misunderstanding of an exception which occurred. sorry.